docs/data/date-pickers/migration-lab/migration-lab.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MUI X - Migration from the lab
+---
+
+# MUI X - Migration from the lab
+
+<p class="description">MUI Date and time pickers are now available on X!</p>
+
+## Introduction
+
+This is a reference for migrating your site's pickers from `@mui/lab` to `@mui/x-date-pickers` or `@mui/x-date-pickers-pro`.
+This migration should be easy
```

Comment:
Maybe better to explain why it will be easy

```suggestion
This migration only concerns packages and should do not affect the behavior of the components in your app. 
```